pysoem/pysoem.pyx

```diff
@@ -718,7 +722,10 @@ cdef class CdefSlave:
         
         cdef cpysoem.ec_errort err
         if cpysoem.ecx_poperror(self._ecx_contextt, &err):
-            self._raise_exception(&err)
+            if err.Etype == cpysoem.EC_ERR_TYPE_EMERGENCY:
+                self._on_emergency(&err)
```

Comment:
@MartinALL91 What if no emergency callback was added, would it be nice to at least trow a warning?

Reply:
Done [0254a91](https://github.com/bnjmnp/pysoem/pull/132/commits/0254a919c22c48a688c66e49d6705def654bd081)